### ISSUE TITLE
Hanging Proposals in: Aitable + Discourse - /project/createProposal can fails to save to mongo + fails to write to Discourse again

### ISSUE THREAD

**Issue by idiom-bytes** [oceanprotocol/oceandao-proposal-portal#85]
### Problem:
There are some issues that may be causing the createProposal route to not complete, failing to save to mongo.
- This causes hanging rows in Airtable + Discourse
- If the proposal gets entered into Discourse but fails to write to Mongo, it starts to create a conflict (backend finds the proposal with the same name), notices the conflict, and fails to post.

## DoD:
- [ ] /project/createProposal functions nearly atomically, and fails to generate hanging bits 99.9% of the time. A way to improve this, would be to do all the upfront checks ahead of time, so nothing is left hanging.

**Comment by idiom-bytes** [comment]
I tried to help DeadmanBob to create a proposal but we kept on failing.

Basically, everything is working:
- Airtable row is created + populated
- Discourse post is created + populated
- Error then gets thrown when trying to write to DB

The request looks correct, but the response is always 502 for him. We have tried both Wallet Connect & MetaMask, and in addition to it, he has tried it in linux, windows, and macosx, with multiple browsers.

My next step would be to create a proposal myself,. If that worked, then I would try to assign myself to deadman's account and go through there.

![new2-bad-gateway-request](https://user-images.githubusercontent.com/69865342/161471678-ef45dfa1-d1d4-4333-9bbb-073f9757b693.png)
